cds_migrator_kit/factory.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
```

Comment:
You can just use `invenio-app` instead of re-creating this one: https://github.com/inveniosoftware/invenio-app/blob/master/invenio_app/factory.py